less/select.less

```diff
@@ -22,8 +22,8 @@
 @select-input-border-width:         1px;
 @select-input-height:               36px;
 @select-input-internal-height:      (@select-input-height - (@select-input-border-width * 2));
-@select-input-placeholder:          #aaa;
```

Comment:
If you need a `000` color, please make it only for the unique select, this changes affects the other select items as well

| Before | After |
| --- | --- |
| ![screen shot 2018-11-26 at 10 36 32 am](https://user-images.githubusercontent.com/1847316/49028186-69938080-f167-11e8-908b-39642fe0e301.png) | ![screen shot 2018-11-26 at 10 36 28 am](https://user-images.githubusercontent.com/1847316/49028187-69938080-f167-11e8-8a38-8e75390538a8.png) |